softioc/extension.c

```diff
@@ -226,27 +221,20 @@ static struct PyMethodDef softioc_methods[] = {
     {NULL, NULL, 0, NULL}        /* Sentinel */
 };
 
-#if PY_MAJOR_VERSION >= 3
+
 static struct PyModuleDef softioc_module = {
   PyModuleDef_HEAD_INIT,
     "softioc._extension",
     NULL,
     -1,
     softioc_methods,
 };
-#endif
 
-#if PY_MAJOR_VERSION >= 3
-#  define PyMOD(NAME) PyObject* PyInit_##NAME (void)
-#else
-#  define PyMOD(NAME) void init##NAME (void)
-#endif
+
+#define PyMOD(NAME) PyObject* PyInit_##NAME (void)
+
 
 PyMOD(_extension)
```

Comment:
Nah, don't use this ugly `PyMOD` macro, we don't need it anymore.  Just write
```python
PyObject *PyInit__extension(void)
```
here instead (and delete the `PyMOD` definition, of course).

Reply:
Done.